### PR TITLE
finished assessment

### PR DESCRIPTION
**Pull request by LaurisaNeuwirth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework for creatign new projects in Ruby
```
Comment (GA-MEB) [inline README.md:9]: This is true, but Rails isn't used for just any projects - it's specifically used for building back-end and full-stack web applications. Rails provides us with architecture and pre-existing files that we can inherit from, and this allows us to spin up new applications very quickly.

File: README.md [hunk 2]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```
Comment (GA-MEB) [inline README.md:12]: Technically, the request gets parsed before any of the controller actions are run. Otherwise correct.

File: README.md [hunk 3]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+
 1. Describe the flow of a PATCH request as it goes through the app.
+The client asks the server to create or add something.
```
Comment (GA-MEB) [inline README.md:15]: Yes.

File: README.md [hunk 4]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+The server asks the appropriate controller.
```
Comment (GA-MEB) [inline README.md:16]: Yes.

File: README.md [hunk 5]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+That controller parses the request to the proper model.
```
Comment (GA-MEB) [inline README.md:17]: The controller invokes methods on the models, causing the database to be manipulated in response to a request.

File: README.md [hunk 6]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+The model interacts with the database and makes the change. The model sends the controller a confirmation message.
```
Comment (MattB-GA-Boston) [inline README.md]: Sometimes, but it's more common that they send some data.

File: README.md [hunk 7]
```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
...
+The controller parses that response in a message to the server.
```
Comment (GA-MEB) [inline README.md:19]: I don't think `parses` is the right word. The model's methods return data, and the controller takes that data and use it to build a response, which gets sent back to the server.

File: README.md (hunk 6 above)
Comment (GA-MEB) [inline README.md:18]: Correct.

**Comment by GA-MEB** [comment]
2 of 3 (-0.5 on Q1; -0.25 on Q2, Q3)

Nice work, Laurissa!

**Pull request closed without merging**